src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -57,6 +58,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         FileOpenPicker picker;
 
         //TODO: SETTINGS UI Commandline handling should be robust and intelligent
+        _State.WindowRoot().TryPropagateHostingWindow(picker); // if we don't do this, there's no HWND for it to attach to
         picker.ViewMode(PickerViewMode::Thumbnail);
         picker.SuggestedStartLocation(PickerLocationId::ComputerFolder);
         picker.FileTypeFilter().ReplaceAll({ L".bat", L".exe" });
```

Comment:
You could probably remove this TODO comment. Feel free to add any other file type filters 😊

Reply:
No, this is neither robust nor intelligent _and_ adding more file extension filters will not help it do so 😉